deploy/lvm-operator.yaml

```diff
@@ -626,6 +626,9 @@ rules:
   - apiGroups: [""]
     resources: ["nodes"]
     verbs: ["get", "list", "watch"]
+  - apiGroups: [""]
+    resources: ["pods"]
```

Comment:
Why is the access required to pod resource?

Reply:
it needs because the csi-resize needs to expand the filesystem that pod is using.